python/lsst/afw/image/maskedImage/maskedImageContinued.py

```diff
@@ -31,6 +31,9 @@
 from .maskedImage import MaskedImageI, MaskedImageF, MaskedImageD, MaskedImageU, MaskedImageL
 
 
+VariancePixel = np.float32
```

Comment:
It's not the best, but doing anything else is surprisingly tricky.  As you know, we can't easily extract the `MaskPixel` equivalent in the pybind11 level from `pybind11::dtype::of<MaskPixel>()`, because NumPy doesn't [always] treat `int` and `int32_t` as equivalent, presumably because the latter happens to be `long` [sometimes].  What you don't know is that that doesn't work for `VariancePixel` either, because it turns out that `np.float32` is not actually a `dtype` object - it's a thing you can construct a `dtype` *from* (and hence pass most places a `dtype` is expected), but it's actually something else that I don't immediately see how to obtain in pybind11.